hub/core/tests/test_compression.py

```diff
@@ -32,6 +32,9 @@
 image_compressions.remove("apng")
 image_compressions.remove("dcm")
 
+# filtering out all the READONLY files from the image_compression
```

Comment:
This comment is not correct. You can remove it.

Reply:
Ok